.env.example

```diff
@@ -1,6 +1,11 @@
-# Service credentials
-DISCOVERY_USERNAME=<username>
-DISCOVERY_PASSWORD=<password>
+# You need to provide either username and password
+DISCOVERY_USERNAME=
+DISCOVERY_PASSWORD=
+# OR IAM API key and URL
+DISCOVERY_IAM_APIKEY=
+DISCOVERY_IAM_URL=https://iam.bluemix.net/identity/token
+
 
 ENVIRONMENT_ID=<environment>
 COLLECTION_ID=<collection>
+DISCOVERY_URL=https://gateway.watsonplatform.net/discovery/api
```

Comment:
I need a default value so that it doesn't break when users are testing the app.

The README describes the parameters that are required and `url` is one of them so hopefully the users will update the URL